parsers/verilog.c

```diff
@@ -1486,7 +1513,12 @@ static void findTag (tokenInfo *const token)
 	{
 		tagNameList (token, skipWhite (vGetc ()));
 	}
-	else if (token->kind != K_UNDEFINED && token->kind != K_IGNORE)
+	else if (token->kind == K_UNDEFINED)
+	{
+		if (isIdentifier(token))	/* user defined type */
+			tagNameList (token, skipWhite (vGetc ()));
+	}
+	else
```

Comment:
Comparing with the original code, the condition `(token->kind != K_IGNORE)` is gone.
Is this o.k.?

Reply:
I withdraw the above comment. The condition is already handled in the new code. Ignore this.